backend/api/v1/auth_service.go

```diff
@@ -730,7 +728,7 @@ func validateEmail(email string) error {
 
 const (
 	// issuerName is the name of the issuer of the OTP token.
-	issuerName = "Bytebase"
+	issuerName = "bytebase"
```

Comment:
Any reason to change this?

Reply:
> Any reason to change this?

It will be a part of url. And url is case-insensitive.